3-0-libraries.md

```diff
@@ -165,36 +191,37 @@ made previously.
   - `PROJ_0002_Br_P_PE_300_EX`
   - `PROJ_0002_Ly_R_PE_300_EX`
 1. From the toolbar, select _Make dilutions_.
-1. Enter the concentrations of the dilutions (use any number you wish).
-1. Ether the current date for the creation date.
+1. Enter the idilution information:
+  * _Conc._: (use any number you wish)
+  * _Creation Date_: (use the current date)
 1. Click _Save_.
 
-At present, Dilutions cannot be saved in Boxes.
 
 <a name="pools"  href="#" id="toplink">top</a>
 
 # 6. Creating Pools
 
 Pools are the last step for libraries before sequencing and represent the
 entity that is loaded onto the flowcell lane or SMRTcell. A "pool" can have one or more
-library dilutions in it. They are equivalent to "worksets" in Geospiza, but are
-required for every lane of sequencing.
+library dilutions in it. Every lane of sequencing contains only one pool. 
 
 <img src="pics/flow-pool.svg"/>
 
 Here we will all all of the dilutions we added previously to make a single pool
```

Comment:
Could you fix the typo here while you're at it? "Here we will **pool** ~~all~~ all of the dilutions..."